docs/bos/queryapi/how-works.md

```diff
@@ -34,6 +34,56 @@ An in-depth, detailed overview of the QueryApi components:
 - **Database:** a Postgres database where the developer's indexer data is stored, using a logical DB per user, and a logical schema per indexer function.
 - **API:** a Hasura server running on Google Cloud Platform exposes a GraphQL endpoint so users can access their data from anywhere.
 
+
+## Provisioning
+
+In summary, QueryAPI provisioning consists of the following steps:
+
+1. Create `database` in [Postgres](#postgres), named after account, if necessary
+2. Add `database` to Hasura
+3. Create `schema` in `database`
+4. Run DDL in `schema`
+5. Track all tables in `schema` in [Hasura](#hasura)
+6. Track all foreign key relationships in `schema` in [Hasura](#hasura)
+7. Add all permissions to all tables in `schema` for account
+
+:::note
+This is the workflow for the initial provisioning. Nothing happens for the remaining provisions, as QueryAPI checks if it has been provisioned, and then skips these steps.
+:::
+
+:::info
+To check if an Indexer has been provisioned, QueryAPI checks if both the `database` and `schema` exist. This check is what prevents the app from attempting to provision an already provisioned indexer.
+:::
+
+## Historical backfill
```

Comment:
We should move this above `## Provisioning` - the subsections below are part of the above but this makes it seems like its related to historical backfill